.codecov.yml

```diff
@@ -0,0 +1,20 @@
+codecov:
+  require_ci_to_pass: yes
+
+coverage:
+  precision: 2
+  round: down
+  range: "70...100"
+
+parsers:
+  gcov:
+    branch_detection:
+      conditional: yes
+      loop: yes
+      method: no
+      macro: no
+
+comment:
+  layout: "reach,diff,flags,files,footer"
+  behavior: default
+  require_changes: no
```

Comment:
please set this to `true`, otherwise we risk to be flooded by coverage comments in some PR. [Ref](https://docs.codecov.io/docs/pull-request-comments#requiring-changes)